hironx_ros_bridge/scripts/hironx.py

```diff
@@ -56,6 +56,7 @@
     parser.add_argument('--modelfile', help='robot model file nmae')
     parser.add_argument('--robot', help='robot modlule name (RobotHardware0 for real robot, Robot()')
     args, unknown = parser.parse_known_args()
+    unknown = [u for u in unknown if u[:2] != '__'] # filter out ros arguments
```

Comment:
https://github.com/start-jsk/rtmros_hironx/pull/316 runs hironx.py with ros arguments and that's causes unexpected behavior around https://github.com/start-jsk/rtmros_hironx/pull/315/files#diff-e6c464699b21a40e8b15aeb08c09a80dL70